### ISSUE TITLE
Bug: ToggleButton "off" state text color does not meet WCAG AA min contrast ratio

### ISSUE THREAD

**Issue by hbuchel** [aws-amplify/amplify-ui#1825]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Other

### How is your app built?

Any

### Please describe your bug.

The toggle button in the "off" state does not meet WCAG AA color contrast requirements of 4.5:1

<img width="139" alt="Screen Shot 2022-05-04 at 10 49 02 AM" src="https://user-images.githubusercontent.com/376920/166707458-c2dd6021-5c67-49b8-b4e9-5f3645a91b7c.png">

<img width="349" alt="Screen Shot 2022-05-04 at 10 35 02 AM" src="https://user-images.githubusercontent.com/376920/166707355-932c8c85-0bdc-4d37-951f-b19273135a68.png">
.

### What's the expected behaviour?

Color contrast should pass 4.5:1 criteria. I recommend we use the default Button text color. 

<img width="139" alt="Screen Shot 2022-05-04 at 10 50 26 AM" src="https://user-images.githubusercontent.com/376920/166707973-f3f1e9c6-b9b4-4e6b-8162-36b97459a5f2.png">

Currently, the Toggle button looks like a disabled button (not interactive, readonly), but really it is in the "off" state.
 

### Help us reproduce the bug!

https://ui.docs.amplify.aws/components/togglebutton

### Code Snippet

```typescript
// Put your code below this line.

```
